components/scream/scripts/scripts-tests

```diff
@@ -360,6 +360,27 @@ class TestTestAllScream(TestBaseOuter.TestBase):
         else:
             self.skipTest("Skipping full run")
 
+    def test_run_diff_opt_not_captured(self):
```

Comment:
Coluld you add a 1 line comment for each of these fcns, explaining what they are testing?

Reply:
Yes